frontend/micro-ui/web/micro-ui-internals/packages/css/src/pages/employee/index.scss

```diff
@@ -373,7 +373,7 @@
 @media (min-width: 640px) {
   .employee {
     .ground-container {
-      @apply block ml-md;
```

Comment:
@prachirane-eGov these styles are common throughout the application. So this will affect other inboxes and wherever this class is used... So pls make sure this doesn't break other inboxes... You can apply additional styling by creating another scss file